src/main/java/Brad/Exceptions/dataCorruptedException.java

```diff
@@ -0,0 +1,5 @@
+package Brad.Exceptions;
+
```

Comment:
Can include methods that display error messages in the exceptions as well, exceptions are classes as well!